src/pudl/metadata/fields.py

```diff
@@ -368,9 +368,13 @@
         "description": "Account balance at end of year.",
         "unit": "USD",
     },
-    "ferc_account_description": {
-        "type": "string",
+    "energy_mwh": {
+        "type": "number",
+        "unit": "MWh",
+        "description": "Sources and uses of energy in MWh.",
     },
+    "energy_source_type": {"type": "string", "description": "Type of energy source."},
```

Comment:
This is not a helpful description. Does it mean Coal? Natural gas? Fossil? Renewable? Imported or domestic? Electrochemical vs. physical? It seems to be related to `energy_source_code` which it is not. Give some examples from and/or context for the classification system it is referring to, or maybe it should have an ENUM constraint?

Reply:
fixed!